src/Column.luna

```diff
@@ -149,6 +148,10 @@ class Column:
         typedCol = TypedColumn typeTag . fromChunks field.ptr chunks.ptr
         typeTag.makeColumn typedCol
 
+    def shortRep: case self of
+        Column: "Column"
```

Comment:
Is it possible to include the type of the column here? I feel like that's information I'd want.

Reply:
Yes, we have `self.field` with the type and name information. Having column name would be nice as well. Perhaps both should be included in `Field`'s `shortRep` and here we could just reuse it.